ui/situations/OBADiversionViewController.m

```diff
@@ -134,6 +134,7 @@ - (MKOverlayRenderer *)mapView:(MKMapView *)mapView rendererForOverlay:(id<MKOve
         overlayView = _routePolylineRenderer;
     }
 
+    // TODO: can't return nil here, MKMapViewDelegate says NONNULL
```

Comment:
MKOverlayRenderer doesn't have any simple initializers. Rethinking this method a bit.